ipywidgets/widgets/widget_upload.py

```diff
@@ -16,12 +17,107 @@
 from .widget_button import ButtonStyle
 from .widget import register, widget_serialization
 from .trait_types import InstanceDict
+from traitlets.utils.bunch import Bunch
```

Comment:
I think you can just do `from traitlets import Bunch` to not rely on the internal traitlets structure.